android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -240,4 +240,14 @@ public void setOptOut(Boolean enabled){
             analytics.setAppOptOut(enabled);
         }
     }
+
+    @ReactMethod
+    public void setAppName(String trackerId, String appName){
+        GoogleAnalytics analytics = getAnalyticsInstance();
+
+        if (analytics != null)
+        {
+            analytics.setAppName(appName);
```

Comment:
This method is incorrect, the `setAppName` method is available on the Tracker, not on the Analytics-instance. See another method in the same class, for example `setAnonymizeIp`, on how to retrieve the tracker and call a method on it.